index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <div class="wrapper">
+      <nav id="navbar">
+        <header>
+          <h1>JS Documentation</h1>
+        </header>
+        <div class="nav-wrapper">
+          <ul>
+            <li><a class="nav-link" href="#Introduction">Introduction </a></li>
+            <li>
+              <a class="nav-link" href="#what_you_should_already_know"
+                >what you should already know</a
+              >
+            </li>
+            <li>
+              <a class="nav-link" href="#JavaScript_and_Java"
+                >JavaScript and Java</a
+              >
+            </li>
+            <li><a class="nav-link" href="#Hello_world">Hello world</a></li>
+            <li><a class="nav-link" href="#Variables">Variables</a></li>
+          </ul>
+        </div>
+      </nav>
+      <main id="main-doc">
+        <section id="Introduction" class="main-section">
+          <header>Introduction</header>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+            Necessitatibus veniam dolores incidunt. Beatae molestias labore
+            perferendis facere? Libero, aut aperiam?
+          </p>
+        </section>
+        <section id="what_you_should_already_know" class="main-section">
+          <header>what you should already know</header>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+            Necessitatibus veniam dolores incidunt. Beatae molestias labore
+            perferendis facere? Libero, aut aperiam?
+          </p>
+        </section>
+        <section id="JavaScript_and_Java" class="main-section">
+          <header>JavaScript and Java</header>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+            Necessitatibus veniam dolores incidunt. Beatae molestias labore
+            perferendis facere? Libero, aut aperiam?
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+        </section>
+        <section id="Hello_world" class="main-section">
+          <header>Hello world</header>
```

Comment:
- [ ] Please use heading tags `<h1>`, `<h2>`, `<h3>`  `<h4>`, `<h5>`, or `<h6>` for they are more semantically fit for this part. Using semantic elements helps for SEO, website accessibility and many more. You can also refer the following link on semantic elements [here](https://www.w3schools.com/html/html5_semantic_elements.asp)